tiatoolbox/data/remote_samples.yaml

```diff
@@ -29,6 +29,8 @@ files:
     url: [*wsis, "CMU-1-Small-Region.ome.tiff"]
   two-tiled-pages:
     url: [*wsis, "two-tiled-pages.tiff"]
+  non-tiled-ventana-tif:
```

Comment:
```suggestion
  ventana-tif:
```

Reply:
will change the names accordingly in all files I touched